test/unit/src/common/selectors/t_selectors.js

```diff
@@ -0,0 +1,53 @@
+import expect from 'expect';
+
+import { hasNoRegisteredNames } from '../../../../../src/common/selectors';
+
+describe('hasNoRegisteredNames', function() {
+
+  const stateWithNoName = {
+    snaps: {
+      snaps: [{}, {}]
+    }
+  };
+
+  const stateWithName = {
+    snaps: {
+      snaps: [{
+        store_name: 'bsi-test-ii'
+      }, {}]
+    }
+  };
+
+  // XXX must run first in suite to avoid memoizing a selector and being
+  // off-by-one in recomputations()
+  context('is a memoized selector', function() {
```

Comment:
I've tried to make this clearer, but the computations count is dependent on tests which ran before even if you reset the count, because the selector may be memoized, and if you give it the same state again, the count won't increment.